### PR TITLE
Improve test's readability 

### PR DESCRIPTION
**Pull request by jmilet**
## Description
This PR improves the test's names and includes small changes in order to make the tests easier to read.

In the first instance it was tried to split the tests in several files, but the result didn't seem to be better that the current structure so no changes regarding that have been made.

## Related Issue
https://app.asana.com/0/1200164532266868/1200217893277504

## Motivation and Context
This PR is part of the Mazzaroth project.

## How Has This Been Tested?
Unit test.

## Screenshots (if appropriate):
N/A

### CODE REVIEW

**Review by lightyear15 — DISMISSED** [review]

**Review by jeff1010322 — APPROVED** [review]
Overall looks good.  It think the tests in crypto could use a similar update:

https://github.com/kochavalabs/mazzaroth-js/blob/develop/test/crypto/ecc-ed25519.js

```
ed25519 ecc
    sign
      ✓ returns a buffer signature
    verify
      ✓ does not verify with bad inputs.
    sign/verify
      ✓ should sign and verify message
      ✓ should sign and verify message2
      ✓ Generate correct signature based on reference impl.
      ✓ Verifies signature based on a reference impl.
    from private
      ✓ gets the corresponding public key
    generate keys
      ✓ Generates two buffers.
```

**Comment by satboy78** [comment]
@jmilet just a general comment, I'd keep the tense consistent across the board (now sometimes is "should", other time "works")

**Comment by jmilet** [comment]
> @jmilet just a general comment, I'd keep the tense consistent across the board (now sometimes is "should", other time "works")

Done. Thanks.

**Comment by jmilet** [comment]
> Overall looks good. It think the tests in crypto could use a similar update:
> 
> https://github.com/kochavalabs/mazzaroth-js/blob/develop/test/crypto/ecc-ed25519.js

Done. Thanks.

File: test/client/node-client.js [hunk 1]
```diff
@@ -21,41 +42,22 @@ function fakeSign (ex1, ex2) {
...
-    }
-  }
-  const authority = {
-    enum: 0,
-    value: ''
-  }
-  describe('construction', () => {
-    it('should pass values', () => {
+  describe('client construction', () => {
+    it('should pass the given parameters', () => {
```
Comment (satboy78) [inline test/client/node-client.js:46]: should :)

File: test/client/node-client.js [hunk 2]
```diff
@@ -21,41 +42,22 @@ function fakeSign (ex1, ex2) {
...
       const client = new NodeClient(defaultRoute, 'ff01', 'a')
       expect(client.host).to.equal(defaultRoute)
       expect(client.privateKey).to.deep.equal(Buffer.from([255, 1]))
       expect(client.sign).to.equal('a')
     })
 
-    it('clean up host with slash', () => {
+    it('should clean up host with slash', () => {
```
Comment (satboy78) [inline test/client/node-client.js:53]: should :)

File: test/client/node-client.js [hunk 3]
```diff
@@ -21,41 +42,22 @@ function fakeSign (ex1, ex2) {
...
       let client = new NodeClient(defaultRoute + '/')
       expect(client.host).to.equal(defaultRoute)
       client = new NodeClient('http://localhost////')
       expect(client.host).to.equal('http://localhost')
     })
 
-    it('accept buffer for key', () => {
+    it('should accept buffer for key', () => {
```
Comment (satboy78) [inline test/client/node-client.js:60]: should :)

File: test/client/contract-client.js [hunk 4]
```diff
@@ -329,20 +339,20 @@ describe('contract calls', () => {
 })
 
 describe('readonly calls', () => {
-  it('read only calls should exist', () => {
+  it('read only validate_signature should exist', () => {
```
Comment (satboy78) [inline test/client/contract-client.js:342]: should :)

File: test/client/utils.js [hunk 5]
```diff
@@ -55,15 +57,15 @@ function getMockClient () {
 }
 
 describe('RunExecutionPlan', () => {
-  it('should throw with bad args', () => {
+  it('should throw exception with bad args', () => {
```
Comment (satboy78) [inline test/client/utils.js:60]: should :)

File: test/client/utils.js [hunk 6]
```diff
@@ -55,15 +57,15 @@ function getMockClient () {
...
     const plan = {
       asdf: 'asdf',
       calls: []
     }
     expect(() => RunExecutionPlan(plan, 'asdf', function () {}, getMockClient())).to.throw()
   })
 
-  it('should not throw with valid plan', () => {
+  it('should not throw exception with valid plan', () => {
```
Comment (satboy78) [inline test/client/utils.js:68]: should :)

File: test/client/utils.js [hunk 7]
```diff
@@ -75,7 +77,7 @@ describe('RunExecutionPlan', () => {
     expect(() => RunExecutionPlan(plan, 'asdf', function () {}, getMockClient())).to.not.throw()
   })
 
-  it('should throw with empty calls array', () => {
+  it('should throw exception with empty calls array', () => {
```
Comment (satboy78) [inline test/client/utils.js:80]: should :)

File: test/client/node-client.js (hunk 1 above)
Comment (jmilet) [reply test/client/node-client.js]: Done. Thanks.

File: test/client/node-client.js (hunk 2 above)
Comment (jmilet) [reply test/client/node-client.js]: Done. Thanks.

File: test/client/node-client.js (hunk 3 above)
Comment (jmilet) [reply test/client/node-client.js]: Done. Thanks.

File: test/client/utils.js (hunk 5 above)
Comment (jmilet) [reply test/client/utils.js]: Done. Thanks.

File: test/client/utils.js (hunk 6 above)
Comment (jmilet) [reply test/client/utils.js]: Done. Thanks.

File: test/client/utils.js (hunk 7 above)
Comment (jmilet) [reply test/client/utils.js]: Done. Thanks.

File: test/client/contract-client.js (hunk 4 above)
Comment (jmilet) [reply test/client/contract-client.js]: Done. Thanks.

**Review by jeff1010322 — APPROVED** [review]

**Review by lightyear15 — APPROVED** [review]

**Pull request merged**